tflint/runner_test.go

```diff
@@ -907,13 +908,31 @@ resource "null_resource" "test" {
 }`,
 			Expected: true,
 		},
+		{
+			Name: "invalid reference",
+			Content: `
+resource "null_resource" "test" {
+	key = invalid
+}`,
+			Expected: false,
+			Error:    errors.New("Invalid reference: A reference to a resource type must be followed by at least one attribute access, specifying the resource name."),
```

Comment:
> __[Golint]__ error strings should not be capitalized or end with punctuation or a newline ([link](https://sider.review/gh/repos/71487396/pulls/443#issue-4579302))

Since this error is returned from Terraform, there is nothing we can do here.